iree/turbine/kernel/wave/utils/mma_utils.py

```diff
@@ -67,7 +67,18 @@ def is_mma(node):
         acc_shape = custom.acc_type.symbolic_shape
 
         try:
-            k = ((set(lhs_shape) & set(rhs_shape)) - set(acc_shape)).pop()
+            common_dims = (set(lhs_shape) & set(rhs_shape)) - set(acc_shape)
```

Comment:
wouldn't `- set(acc_shape)` get rid of the batch dims?
```
(B, M, K) & (B, K, N) - (B, M, N)
= (B,K) - (B, M ,N)
= (K)
```

Reply:
I think the name is the problem here, it should be 
```
reduction_dim_candidates = (set(lhs_shape) & set(rhs_shape)) - set(
                acc_shape
            )
```
In this case it does the right thing in your example above. But also for the case where
```
(B, M, K) & (B, K, N) - (M, N) -> (B, K)
```

Reply:
In what cases do we expect the accumulator to be (M, N) as opposed to (B, M, N)?

Reply:
The test has this as an example. In general, not sure how frequent this might be as a use case.